entries/w/wise.com.json

```
@@ -7,9 +7,8 @@
       "custom-software"
     ],
     "documentation": "https://wise.com/help/12/managing-your-profile/2932125",
```

```suggestion
    "documentation": "https://wise.com/help/12/managing-your-profile/2932125",
    "notes": "SMS 2FA must be enabled before software methods can be enabled.",
```
Although Google Voice may be able to be used for SMS 2FA, the documentation page mentions that SMS 2FA must be enabled before the app can be used for 2FA, so a `notes` field is still required.